js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -1615,7 +1615,7 @@ updateFirstLoginHelp=Update profile on first login
 deleteGroup=Delete group
 eventTypes.VERIFY_EMAIL_ERROR.description=Verify email error
 close=Close
-usersDNHelp=Full DN of LDAP tree where your users are. This DN is the parent of LDAP users. It could be for example 'ou\=users,dc\=example,dc\=com' assuming that your typical user will have DN like 'uid\='john',ou\=users,dc\=example,dc\=com'.
+usersDNHelp=Full DN of the LDAP tree where your users are. This DN is the parent of LDAP users. It could be for example 'ou\=users,dc\=example,dc\=com' assuming that your typical user will have DN like 'uid\='john',ou\=users,dc\=example,dc\=com'.
```

Comment:
```suggestion
usersDNHelp=Full DN of the LDAP tree where your users exist. This DN is the parent of LDAP users. It could be for example 'ou\=users,dc\=example,dc\=com' assuming that your typical user will have DN such as 'uid\='john',ou\=users,dc\=example,dc\=com'.
```